src/pyscipopt/scip.pyx

```diff
@@ -1008,6 +1010,100 @@ cdef class Model:
         free(varexprs)
         return PyCons
 
+    def _addGenNonlinearCons(self, ExprCons cons, **kwargs):
```

Comment:
I'll have to believe you on this function...

Reply:
haha, this is basically a copy-paste of the one in CSIP